enhancements/sig-policy/99-policy-placement-strategy/README.md

```diff
@@ -0,0 +1,302 @@
+    Details         []*DetailsPerTemplate         `json:"details,omitempty"`
+  }
+  ```
+
+#### Story 2
+
+As a system administrator, I want to use a placement `DecisionStrategy` with policies to control the order of the
+updates.
+
+- **Summary**
+
+  - Use the `Placement` library to gather the list of clusters and iterate over them.
+
+- **Details**
+
+  - Make `Placement` the primary placement resource and parse `PlacementRule` decisions into the `Placement` struct
+    instead of the `PlacementRule` struct.
+  - Using the `Placement` library and implementing a `ClusterRolloutStatusFunc`, iterate over all clusters as usual
+    using the `All` rollout strategy.
+
+- **Snippet (untested)**
+
+  ```golang
+  // common.go
+  type placementDecisionGetter struct {
+    c client.Client
+  }
+
+  func (pd placementDecisionGetter) List(selector labels.Selector, namespace string) ([]*clusterv1beta1.PlacementDecision, error) {
+    pdList := &clusterv1beta1.PlacementDecisionList{}
+    lopts := &client.ListOptions{
+      LabelSelector: selector,
+      Namespace:     namespace,
+    }
+
+    err := pd.c.List(context.TODO(), pdList, lopts)
+    if err != nil {
+      return nil, err
+    }
+
+    pdPtrList := []*clusterv1beta1.PlacementDecision{}
+    for _, pd := range pdList.Items {
+      pdPtrList = append(pdPtrList, &pd)
+    }
+
+    return pdPtrList, err
+  }
+
+  func GetRolloutHandler(c client.Client, placement *clusterv1beta1.Placement) (*clusterv1alpha1.RolloutHandler, error) {
+    pdTracker := clusterv1beta1.NewPlacementDecisionClustersTracker(placement, placementDecisionGetter{c}, nil, nil)
+
+    _, _, err := pdTracker.Get()
+    if err != nil {
+      log.Error(err, "Error retrieving PlacementDecisions from tracker")
+    }
+
+    return clusterv1alpha1.NewRolloutHandler(pdTracker)
+  }
+
+  var GetClusterRolloutStatus clusterv1alpha1.ClusterRolloutStatusFunc = func(clusterName string) clusterv1alpha1.ClusterRolloutStatus {
+    // Fetch and return the rollout status and the last transition time from the status in the replicated policy
+
+    return clusterv1alpha1.ClusterRolloutStatus{
+      Status: "<rollout-status>",
+      LastTransitionTime: "<transition-time>",
+    }
+  }
+  ```
+
+  ```golang
+  rolloutHandler := common.GetRolloutHandler(client, placement)
+  strategy, rolloutResult, err := rolloutHandler.GetRolloutCluster(clusterv1alpha1.All, common.GetClusterRolloutStatus)
+
+  // ... Use rolloutResults for propagation
+  ```
+
+#### Story 3
+
+As a system administrator, I want policies to be rolled out only when the policy on previous clusters show as
+"Compliant" and control their concurrency and priority.
+
+- **Summary**
+
+  - Only continue policy deployment once the previously deployed clusters show as "Compliant".
+
+- **Details**
+
+  - Update the `PlacementBinding` CRD to contain the `RolloutStrategy` struct. (See the
```

Comment:
I think its better to define the RolloutStrategy under the Policy->Spec not under the PlacementBinding cause a policy can have multiple placementBinding so as a user should I repeated RolloutStrategy across all placementBindings? 
I guess defining the RolloutStrategy under PlacementBinding  will be challenging to handle.

Reply:
I was working on updating this as you were reviewing 🙂 We came to the same conclusion and have updated the enhancement to specify the RolloutStrategy on the Policy rather than the PlacementBinding.